Wiki/attributes.md

```diff
@@ -22,6 +22,7 @@ Labels are also searchable, enhancing note retrieval.
 ### Common Labels for Advanced Configuration
 
 - **`disableVersioning`**: Disables automatic versioning, ideal for large, unimportant notes like script libraries
+- **`versioningLimit`**: Used to limit the number of individual notes.
```

Comment:
Is this correct? Shouldn't it be the limit of revisions of individual notes?

Reply:
Modified